docs/latest/concepts/islands.md

```diff
@@ -78,6 +78,32 @@ export default function Home() {
 }
 ```
 
+You can also create shared components in your `components/` directory, which can
+be used in both static content and interactive islands.
+
+```tsx islands/my-island.tsx
+import { useSignal } from "@preact/signals";
+import { ComponentChildren } from "preact";
+import Card from "../components/Card.tsx";
+import Button from "../components/Button.tsx";
+
+interface Props {
+  children: ComponentChildren;
+}
+
+export default function MyIsland({ children }: Props) {
+  const count = useSignal(0);
+
+  return (
+    <Card>
+      Counter is at {count}.{" "}
+      <Button onClick={() => (count.value += 1)}>+</Button>
```

Comment:
I file like we should add a sentence that this click handler will only fire when this component is inside `./islands` or referenced inside an island. I'm worried that the component name here suggests that this component would be treated as an island.

Reply:
Thanks @marvinhagemeister 

I hoped the positioning in the docs would have helped clarify that, but I've added a sentence as you mention to make sure it's explicitly known. Let me know what you think.